ivy-prescient.el

```diff
@@ -33,55 +33,13 @@
 
 ;;;; User options
 
-(defcustom ivy-prescient-excluded-commands
-  '(counsel-ag
-    counsel-expression-history
-    counsel-git-grep
-    counsel-grep
-    counsel-mark-ring
-    counsel-minibuffer-history
-    counsel-shell-command-history
-    counsel-yank-pop
-    swiper)
-  "Commands for which candidates should not be remembered."
-  :group 'prescient
-  :type '(list symbol))
-
-(defcustom ivy-prescient-sort-commands
-  '(counsel-find-file
-    counsel-find-library)
+(defcustom ivy-prescient-sort-commands '(counsel-find-library)
```

Comment:
Any particular reason to remove `counsel-find-file`, by the way?

Reply:
Not needed. Any ivy session using `read-file-name-internal` as the collection is handled specially. An example of this is that they have sorting enabled by default, even though it isn't explicitly listed in their properties.

Reply:
Cool, sounds good.